src/test/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepositoryTest.java

```diff
@@ -362,6 +374,38 @@ public void whenValidateInvalidCommitCursorsThenException() throws NakadiExcepti
                 });
     }
 
+    @Test
+    public void whenKafkaPublishTimeoutThenCircuitIsOpened() throws Exception {
```

Comment:
@adyach I see. I thought that "timed out" came from actual producer timeout and not from short circuiting. Changing it to something related to short circuiting would help me understand it better 👍